### PR TITLE
remove some extraneous text from schema doc

### PR DESCRIPTION
**Pull request by kamahen**
(This came up in discussion #3734)

### CODE REVIEW

File: kythe/docs/schema/schema.txt [hunk 1]
```diff
@@ -501,8 +501,8 @@ lines owing to heuristics in Clang's comment parser.
 
 [kythe,C++,"Comments document objects."]
 --------------------------------------------------------------------------------
-int v;   //- @"/// An empty class." documents ClassC
-         //- ClassC.node/kind record
+//- @"/// An empty class." documents ClassC
+//- ClassC.node/kind record
```
Comment (shahms) [inline kythe/docs/schema/schema.txt]: I'm not entirely sure why, but apparently this change causes the schema tests to fail, @zrlk ?

Comment (kamahen) [reply kythe/docs/schema/schema.txt]: My guess, based on re-re-re-reading the text above the example, is that the original is a way of getting the two sets of comments to be separate (the `//-` comments for the verifier and the `///` comment for the test). So, I should remove my PR and leave future generations to puzzle this out. ;)

Comment (zrlk) [reply kythe/docs/schema/schema.txt]: Without the `int v;` there, I think Clang will merge these comments with the class comment into a single block.

**Comment by kamahen** [comment]
Based on that, I'll delete this PR.

**Pull request closed without merging**